### PR TITLE
Adding MB limitations to documentation

### PR DESCRIPTION
**Pull request by napetrov**
# Description
adding known  limitations to Model Builders

### CODE REVIEW

File: doc/daal4py/model-builders.rst [hunk 1]
```diff
@@ -83,6 +83,16 @@ You can also use the scikit-learn-style classes ``GBTDAALClassifier`` and ``GBTD
   reg.fit(X, y)
   d4p_predt = GBTDAALRegressor.convert_model(reg).predict(X)
 
+
+Limitations
+------------------
+Model Buildes does support only base inference with prediction and probabilities prediction at this point and functioanlity will be extended going forward.
+This means thet there are following limitations exist:
+- Categorical features are not supported during training and prediction
```
Comment (razdoburdin) [inline doc/daal4py/model-builders.rst:91]: I guess, we shouldn't mention training in context of model builders.

**Review by razdoburdin — APPROVED** [review]

File: doc/daal4py/model-builders.rst [hunk 2]
```diff
@@ -83,6 +83,16 @@ You can also use the scikit-learn-style classes ``GBTDAALClassifier`` and ``GBTD
   reg.fit(X, y)
   d4p_predt = GBTDAALRegressor.convert_model(reg).predict(X)
 
+
+Limitations
+------------------
+Model Buildes does support only base inference with prediction and probabilities prediction at this point and functioanlity will be extended going forward.
```
Comment (aepanchi) [inline doc/daal4py/model-builders.rst:89]:
```suggestion
Model Builders support only base inference with prediction and probabilities prediction. The functionality is to be extended.
```

File: doc/daal4py/model-builders.rst [hunk 3]
```diff
@@ -83,6 +83,16 @@ You can also use the scikit-learn-style classes ``GBTDAALClassifier`` and ``GBTD
...
+This means thet there are following limitations exist:
```
Comment (aepanchi) [inline doc/daal4py/model-builders.rst:90]:
```suggestion
Therefore, there are following limitations:
```

Comment (aepanchi) [reply doc/daal4py/model-builders.rst:90]: Add a blank line after this one to make a bulleted list.

File: doc/daal4py/model-builders.rst [hunk 4]
```diff
@@ -83,6 +83,16 @@ You can also use the scikit-learn-style classes ``GBTDAALClassifier`` and ``GBTD
...
+- Categorical features are not supported for conversion and prediction
```
Comment (aepanchi) [inline doc/daal4py/model-builders.rst:91]:
```suggestion
- The categorical features are not supported for conversion and prediction. 
```

File: doc/daal4py/model-builders.rst [hunk 5]
```diff
@@ -83,6 +83,16 @@ You can also use the scikit-learn-style classes ``GBTDAALClassifier`` and ``GBTD
...
+- Multioutput models are not supported for conversion and prediction
```
Comment (aepanchi) [inline doc/daal4py/model-builders.rst:92]:
```suggestion
- The multioutput models are not supported for conversion and prediction.
```

File: doc/daal4py/model-builders.rst [hunk 6]
```diff
@@ -83,6 +83,16 @@ You can also use the scikit-learn-style classes ``GBTDAALClassifier`` and ``GBTD
...
+- Tree SHAP calculations are not supported
```
Comment (aepanchi) [inline doc/daal4py/model-builders.rst:93]:
```suggestion
- The tree SHAP calculations are not supported.
```

File: doc/daal4py/model-builders.rst [hunk 7]
```diff
@@ -83,6 +83,16 @@ You can also use the scikit-learn-style classes ``GBTDAALClassifier`` and ``GBTD
   reg.fit(X, y)
   d4p_predt = GBTDAALRegressor.convert_model(reg).predict(X)
 
+
+Limitations
+------------------
+Model Builders support only base inference with prediction and probabilities prediction. The functionality is to be extended.
+Therefore, there are following limitations:
```
Comment (aepanchi) [inline doc/daal4py/model-builders.rst:90]:
```suggestion
Therefore, there are the following limitations:
```

**Review by aepanchi — APPROVED** [review]
Small suggestion, the rest looks good!

**Pull request merged**